test/test-v8debugapi.js

```diff
@@ -319,9 +321,39 @@ describe('v8debugapi', function() {
     '{f: process.env}',
     '1,2,3,{f:2},4',
     'A[this?this:1]',
-    '[1, 2, 3, 4, x == 1, x === 1, null, undefined]'
+    '[1, 2, 3, 4, x == 1, x === 1, null, undefined]',
+    '[0].values',
+    '[][0]',
+    '[0][' + MAX_INT + ']',
+    '"𠮷".length + (5| "𠮷")',
+    '/ٹوٹ بٹوٹ کے دو مُرغے تھے/',
   ]);
 
+  if (semver.satisfies(process.version, '>=4.0')) {
```

Comment:
The purpose of this test is to ensure that side-effect free ES6 expressions are allowed to be evaluated by the debug agent. We already have tests that ensure invalid grammar is gracefully handled. Adding specific ES6 examples that we expect to fail to parse isn't necessarily something that would provide value to these tests. I am not opposed to adding more examples here, but those aren't an objective for this set of tests.